### PR TITLE
Small Code Organization Refactor

### PR DESCRIPTION
**Pull request by A1Liu**
Moved all 100% database specific stuff to the `crate::db` module. That means moving `sql_types.rs` and `schema.rs`, so that the tree now looks like this:

```bash
├── authorization.rs
├── db
│   ├── mod.rs        <==
│   ├── schema.rs     <== Changes happened to these files
│   └── sql_types.rs  <==
├── github.rs
├── main.rs
├── resolvers
│   └── ...
├── routes
│   └── ...
└── utils.rs
```

### CODE REVIEW

File: src/db/schema.rs [hunk 1]
```diff
@@ -1,6 +1,7 @@
 table! {
+
     use diesel::sql_types::*;
-    use crate::sql_types::*;
+    //use crate::db::sql_types::*;
```
Comment (NicholasLYang) [inline src/db/schema.rs]: We should change this in diesel.toml. Check the `import_types` option

File: src/main.rs [hunk 2]
```diff
@@ -31,8 +31,8 @@ mod github;
 mod resolvers;
 mod routes;
 #[allow(unused_imports)]
-mod schema;
-mod sql_types;
+// mod schema;
```
Comment (NicholasLYang) [inline src/main.rs]: let's delete these

File: src/db/schema.rs (hunk 1 above)
Comment (A1Liu) [reply src/db/schema.rs]: ok, should be good to go now!

File: src/main.rs (hunk 2 above)
Comment (A1Liu) [reply src/main.rs]: Done!

**Comment by A1Liu** [comment]
Closing in favor of #7 , which extends this branch with more wide-reaching refactors.

**Comment by NicholasLYang** [comment]
Eh, let's reopen and merge this first

**Pull request merged**